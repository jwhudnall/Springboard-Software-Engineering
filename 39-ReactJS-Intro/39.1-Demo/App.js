const App = () => {
  return (
    <div>
      <RandomNum />
      <RandomNum />
      <RandomNum />
      <Shiba />
      <Shiba />
      <Doge />
      <Shiba />
      <Shiba />
    </div>
  );
};

ReactDOM.render(<App />, document.getElementById("root"));
