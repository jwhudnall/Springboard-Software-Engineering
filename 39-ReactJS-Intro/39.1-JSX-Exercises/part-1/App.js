const App = () => {
  return (
    <div>
      <FirstComponent />
      <NamedComponent name='James' />
    </div>
  );
};

ReactDOM.render(<App />, document.getElementById("root"));
