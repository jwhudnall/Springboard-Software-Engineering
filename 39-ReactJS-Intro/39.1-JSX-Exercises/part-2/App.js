const App = () => {
  return (
    <div>
      <Tweet
        username='jayjay47'
        name='Jay'
        date={new Date().toDateString()}
        message='Great day to code some stuff!'
      />
      <Tweet
        username='jayjay48'
        name='Jay'
        date={new Date().toDateString()}
        message='Great day to code some stuff!'
      />
      <Tweet
        username='jayjay49'
        name='Jay'
        date={new Date().toDateString()}
        message='Great day to code some stuff!'
      />
    </div>
  );
};

ReactDOM.render(<App />, document.getElementById("root"));
