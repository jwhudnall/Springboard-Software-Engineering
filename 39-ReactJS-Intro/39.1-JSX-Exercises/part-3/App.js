const App = () => {
  return (
    <div>
      <Person name='James' age={35} hobbies={["Coding", "Working Out", "Learning"]} />
      <Person name='Sam' age={30} hobbies={["Reading", "Walking", "Singing"]} />
      <Person
        name='My name is longer than eight'
        age={15}
        hobbies={["Skating", "Running", "Dancing"]}
      />
    </div>
  );
};

ReactDOM.render(<App />, document.getElementById("root"));
