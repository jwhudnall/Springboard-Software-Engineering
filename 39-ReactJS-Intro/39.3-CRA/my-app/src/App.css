h1 {
  color: teal;
}

li {
  color: yellowgreen;
}
