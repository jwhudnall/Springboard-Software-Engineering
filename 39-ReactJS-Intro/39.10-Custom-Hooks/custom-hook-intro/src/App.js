import logo from "./logo.svg";
import "./App.css";
import MoodClicker from "./MoodClicker";
import Counter from "./Counter";
import SignupForm from "./SignupForm";
import DogDetail from "./DogDetail";

function App() {
  return (
    <div className='App'>
      <DogDetail />
      <SignupForm />
      <Counter />
      <MoodClicker />
    </div>
  );
}

export default App;
