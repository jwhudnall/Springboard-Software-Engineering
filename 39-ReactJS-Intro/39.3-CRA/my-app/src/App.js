import React from "react";
import "./App.css";
import items from "./items";
import ShoppingCart from "./ShoppingCart";
import moreItems from "./moreItems";
import Alert from "./Alert";

function App() {
  return (
    <div>
      <Alert variant='success'>
        <h1>Welcome Back!</h1>
      </Alert>
      <Alert variant='danger'>
        <h1>Oh-no!</h1>
      </Alert>
      <ShoppingCart items={items} username='Carly' />
      <ShoppingCart items={moreItems} username='Jin' />
    </div>
  );
}

export default App;
