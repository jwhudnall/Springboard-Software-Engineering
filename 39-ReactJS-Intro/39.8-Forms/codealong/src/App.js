import UserForm from "./UserForm";
import "./App.css";
import ShoppingList from "./ShoppingList";
import SimpleForm from "./SimpleForm";

function App() {
  return (
    <div className='App'>
      {/* <ShoppingList />
      <UserForm /> */}
      <SimpleForm />
    </div>
  );
}

export default App;
