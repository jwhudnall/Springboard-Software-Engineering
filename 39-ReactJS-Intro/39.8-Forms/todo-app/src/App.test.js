import { render, screen } from "@testing-library/react";
import App from "./App";

test("renders", () => {
  render(<App />);
  expect(screen.getByText("Walk the dog.")).toBeInTheDocument();
});

it("matches snapshot", function () {
  const { asFragment } = render(<App />);
  expect(asFragment()).toMatchSnapshot();
});
