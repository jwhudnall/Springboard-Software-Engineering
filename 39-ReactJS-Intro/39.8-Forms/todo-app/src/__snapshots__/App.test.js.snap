// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="App"
  >
    <header
      class="App-header"
    >
      <form>
        <label
          for="task"
        >
          Task
        </label>
        <input
          id="task"
          name="task"
          required=""
          type="text"
          value=""
        />
        <button>
          Add
        </button>
      </form>
      <ul>
        <li>
          Walk the dog.
          <button>
            X
          </button>
        </li>
        <li>
          Take out the Trash.
          <button>
            X
          </button>
        </li>
        <li>
          Study.
          <button>
            X
          </button>
        </li>
      </ul>
    </header>
  </div>
</DocumentFragment>
`;
